src/services/category.js

```
@@ -61,13 +62,32 @@ const categoryService = {
+    const subjectQuery = {
+      category: { $in: categoriesIds }
+    }
+
+    const subjects = await Subject.find(subjectQuery).lean()
+
+    const mappedData = data.map((category) => {
+      const totalOffers = subjects.reduce(
+        (acc, subject) => {
+          if (subject.category.toString() === category._id.toString()) {
+            acc.student += subject.totalOffers.student
+            acc.tutor += acc.tutor + subject.totalOffers.tutor
```

Biome (1.9.4)

[error] Variable appears on both sides of an assignment operation.

This assignment might be the result of a wrong refactoring.
Unsafe fix: Use acc.tutor += subject.totalOffers.tutor instead.


(lint/suspicious/noMisrefactoredShorthandAssign)